static/docs/tutorial/define-ml-pipeline.md

```diff
@@ -120,11 +119,6 @@ $ du -sh .
  41M .
 ```
 
-> Note that DVC tries to use reflinks by default in the platforms that support
-> them (Mac OS or Linux, depending on the file system). Some implementation
-> details (like inodes) might differ, but the overall DVC behavior is the same
-> on those platforms.
```

Comment:
Removed unnecessary note explained in /docs/user-guide/large-dataset-optimization (linked to from this doc)